### DISCUSSION TITLE
Export-DbaInstance, i'll get Could not connect because the maximum number of '1' dedicated administrator connections already exists

### DISCUSSION THREAD

**Discussion by cplex** [dataplat/dbatools#7368] [Q&A]
Hello,
I have tried to use dbatools  "Export-DbaInstance -Exclude Databases -SqlInstance $instancename"  it works except for when i try to export the logins.
Then it hangs all the time (even after restart of the whole server),
It just says  Exporting logins..... and i get a warning about the DAC is already in use but when i check this i can't see that either?
DATE/TIME:	2021-05-24 09:42:56
DESCRIPTION:	Could not connect because the maximum number of '1' dedicated administrator connections already exists. Before a new connection can be made,
the existing dedicated administrator connection must be dropped, either by logging off or ending the process. [CLIENT: 127.0.0.1]
The story behind is this was an SQL 2014 which we upgraded to SQL2019 this weekend and needed "backup" in case of disaster, but it failed, and also the same thing after the upgrade.

**No one is using DAC**  (no rows returened )
SELECT s.session_id ,
       e.name AS Endpoint ,
       s.login_name ,
       s.nt_domain ,
       s.nt_user_name ,
       s.login_time ,
       s.host_name ,
       s.program_name
 FROM sys.dm_exec_sessions s
 INNER JOIN sys.endpoints e
 ON e.endpoint_id = s.endpoint_id
 WHERE e.is_admin_endpoint=1
SELECT * FROM sys.endpoints
**Also allowing remote connections**
name	endpoint_id	principal_id	protocol	protocol_desc	type	type_desc	state	state_desc	is_admin_endpoint
Dedicated Admin Connection	1	1	2	TCP	2	TSQL	0	STARTED	1 (edited) 

On Windows Server 2016, SQL Server 2019 (CU10) , Powershell 5, dbatools version 1.0.149

**Comment by andreasjordan** [comment]
This may answer your question: https://github.com/sqlcollaborative/dbatools/issues/6372
